src/app/header/style.css

```diff
@@ -0,0 +1,115 @@
+.container-header_mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.125rem 1.5rem 0.75rem;
+  gap: 1rem;
+}
+
+.button-header_mobile {
+  background-color: var(--color-primary-dark);
+  border: 1px solid var(--color-primary-darkest);
+  border-radius: 100vh;
+  color: var(--color-neutral-lightest);
+  font-size: var(--font-size-6);
+  font-weight: var(--fw-medium);
+  line-height: 1.2;
+  white-space: nowrap;
+  padding: 0.6rem 1.25rem;
+}
+
+.button-header_mobile:hover {
+  background-color: var(--color-primary-darkest);
+}
+
+.navbar_mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-top: 1px solid var(--color-primary-lightest);
+  box-shadow: 0px 4px 4px rgba(0, 122, 255, 0.15);
+  padding-inline: 1.5rem;
+}
+
+.custom-link_mobile {
+  color: var(--color-primary-darkest);
+  font-size: var(--font-size-6);
+  font-weight: var(--fw-regular);
+  padding-block: 0.5rem;
+}
+
+.custom-link_mobile:hover {
+  text-decoration: underline;
+  text-underline-offset: 0.9em;
+}
+
+.container-header_desktop {
+  display: none;
+}
+
+@media (min-width: 1260px) {
+  .container-header_desktop {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-block: 1rem;
+    box-shadow: 0px 4px 4px rgba(0, 122, 255, 0.15);
+    background-color: var(--color-neutral-lightest);
+  }
+
+  .logo_desktop {
+    min-width: 176px;
+    margin-inline: 1.625em 8.5em;
+  }
+
+  .custom-link_desktop {
+    text-align: center;
+    color: var(--color-primary-darkest);
+    font-size: var(--font-size-6);
+    font-weight: var(--fw-regular);
+    min-width: 140px;
+    padding-block: 0.5rem;
+    border-bottom: 1px solid transparent;
+  }
+
+  .custom-link_desktop:hover {
+    border-bottom: 1px solid var(--color-primary-darkest);
+  }
+
+  .button-header_desktop {
+    background-color: var(--color-primary-dark);
+    border: 1px solid var(--color-primary-darkest);
+    border-radius: 100vh;
+    color: var(--color-neutral-lightest);
+    font-size: var(--font-size-6);
+    font-weight: var(--fw-medium);
+    line-height: 1.2;
+    white-space: nowrap;
+    padding: 0.7rem 1.25rem;
```

Comment:
Parte da estilização do button no desktop e mobile é a mesma. O que for igual, extrair para uma classe compartilhada. @pdrmenezes @KaliniV